searx/engines/opensemantic.py

```diff
@@ -0,0 +1,47 @@
+"""
+Open Semantic Search
+
+ @website    https://www.opensemanticsearch.org/
+ @provide-api yes (https://www.opensemanticsearch.org/dev)
+
+ @using-api  yes
+ @results    JSON
+ @stable     yes
+ @parse      url, title, content, publishedDate
+"""
+from dateutil import parser
+from json import loads
+from searx.url_utils import quote
+
+# If your OSS is hosted on another machine you would need to change
+# your base_url.
+base_url = 'http://localhost:8983/solr/opensemanticsearch/'
```

Comment:
Is this the classical URL given by an installation of Open Semantic Search ?
You should maybe put it in `settings.yml`. This way, the configuration is only done in the configuration file.
(So you can remove the comment maybe)

What do you think ?